src/specifications/profile-resolution/profile-resolution-examples/abc-full_catalog.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../example-checkup.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<catalog id="abc-full_catalog" xmlns="http://csrc.nist.gov/ns/oscal/1.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://csrc.nist.gov/ns/oscal/1.0 ../../../../xml/schema/oscal_catalog_schema.xsd">
+    <metadata>
+        <title>ABC Catalog</title>
+        <last-modified>2019-11-01T10:16:22.939-04:00</last-modified>
+        <version>1.0</version>
+        <oscal-version>1.0-MR2</oscal-version>
```

Comment:
I think we used `1.0.0-milestone2` in existing examples.

Reply:
Okay.